crates/runtime/tests/postgres/mod.rs

```diff
@@ -22,13 +22,15 @@ use arrow::{
 };
 use data_components::postgres::DynPostgresConnectionPool;
 use datafusion::execution::context::SessionContext;
+use serial_test::serial;
 use sql_provider_datafusion::SqlTable;
 
 use crate::init_tracing;
 
 mod common;
 
 #[tokio::test]
+#[serial]
```

Comment:
I would prefer to keep these tests running in parallel - I think that means we need to customize the Docker container runner to specify a port (or maybe pick a random port) so we can run multiple instances at once.